docs/integrations/builtin/trigger-nodes/n8n-nodes-base.telegramtrigger.md

```diff
@@ -18,15 +18,23 @@ You can find authentication information for this node [here](/integrations/built
 For usage examples and templates to help you get started, refer to n8n's [Telegram Trigger integrations](https://n8n.io/integrations/telegram-trigger/){:target=_blank .external-link} page.
 ///
 
-## Updates
-
-- `*`: All updates
-- **message**: Trigger on a new incoming message of any kind- text, photo, sticker, etc
-- **edited_message**: Trigger on a new version of a channel post that's known to the bot and was edited
-- **channel_post**: Trigger on a new incoming channel post of any kind - text, photo, sticker, etc
-- **edited_channel_post**: Trigger on a new version of a channel post that's known to the bot and was edited
-- **inline_query**: Trigger on a new incoming inline query
-- **callback_query**: Trigger on a new incoming callback query
-- **shipping_query**: Trigger on a new incoming shipping query. Only for invoices with flexible price
-- **pre_checkout_query**: Trigger on a new incoming pre-checkout query. Contains full information about checkout
-- **poll**: Trigger on a new poll state. Bots receive only updates about stopped polls and polls which are sent by the bot
+## Events
+
+- **`*`**: Triggers on all updates of any kind.
+- **Callback Query**: Triggers on a new incoming callback query.
+- **Channel Post**: Triggers on a new incoming channel post of any kind, including text, photo, sticker, and so on.
+- **Edited Channel Post**: Triggers on a new version of a channel post that's known to the bot and was edited.
+- **Edited Message**: Triggers on a new version of a message that's known to the bot and was edited.
+- **Inline Query**: Triggers on a new incoming inline query.
+- **Message**: Triggers on a new incoming message of any kind, including text, photo, sticker, and so on.
+- **Poll Change**: Triggers when the poll state changes. Bots receive updates about stopped polls and polls which are sent by the bot.
```

Comment:
In the UI, the option is called 'Poll', is there a reason why you've renamed it this way?